services/py-api/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM python:3.12.6-bullseye AS builder
+LABEL authors="ivanobreshkov"
+
+# https://medium.com/@albertazzir/blazing-fast-python-docker-builds-with-poetry-a78a66f5aed0
+
+RUN pip install poetry
+
+ENV POETRY_NO_INTERACTION=1 \
+    POETRY_VIRTUALENVS_IN_PROJECT=1 \
+    POETRY_VIRTUALENVS_CREATE=1 \
+    POETRY_CACHE_DIR=/tmp/poetry_cache
+
+WORKDIR /app
+
+COPY pyproject.toml poetry.lock README.md ./
+
+RUN poetry install --no-root && rm -rf $POETRY_CACHE_DIR
+
+# The runtime image, used to just run the code provided its virtual environment
+FROM python:3.12.6-slim-bullseye AS runtime
+
+RUN apt-get update && apt-get install -y --no-install-recommends curl && rm -rf /var/lib/apt/lists/*
+
+ENV VIRTUAL_ENV=/app/.venv \
+    PATH="/app/.venv/bin:$PATH"
+
+COPY --from=builder ${VIRTUAL_ENV} ${VIRTUAL_ENV}
+
+WORKDIR /app
+
+COPY src ./src
+COPY .env ./
```

Comment:
Wouldn't these be injected by the compose configuration? I honestly believe that you shouldn't depend on the `.env` file here. If you're spinning up just this image, pass them inline.

Reply:
I think it will be passed by the docker-compose. This PR is only for the docker image itself. I thought of making the docker-compose in another, once we clear this one.

Reply:
Yes, that's what I thought as well. I believe you should get rid of this `COPY` then.